generators/Exercises/Clock.cs

```diff
@@ -65,5 +66,21 @@ private static string RenderConsistencyToAssert(TestMethodBody testMethodBody)
 
             return TemplateRenderer.RenderInline(template, testMethodBody.AssertTemplateParameters);
         }
+
+        private static string RenderEqualToAssert(TestMethodBody testMethodBody)
+        {
+            testMethodBody.AssertTemplateName = "AssertEquals";
+            var ExpectedParameter = testMethodBody.CanonicalDataCase.Input[ParamClock2];
+            var TestedValue = "sut";
+            var expectedEqual = testMethodBody.CanonicalDataCase.Properties[PropertyEqualityExpectance];
+
+            testMethodBody.AssertTemplateParameters = new { ExpectedParameter, TestedValue};
+
+            var template = expectedEqual
+                ? $"Assert.Equal({{{{ ExpectedParameter }}}}, {{{{ TestedValue }}}}); "
+                : $"Assert.NotSame({{{{ ExpectedParameter }}}}, {{{{ TestedValue }}}});";
```

Comment:
NotSame should be NotEqual I think, as NotSame merely check if they are the exact same instance.